### DISCUSSION TITLE
[@types/workerpool] is very old , current workerpool version is 6.2.0

### DISCUSSION THREAD

**Discussion by EngHosam** [DefinitelyTyped/DefinitelyTyped#58219] [Issues with a @types package]
@types/workerpool is very old , current workerpool version is 6.2.0

[a comment by an automated account is not shown]

**Reply by EngHosam** [reply]
@Alorel, @sgkim126, @forivall

**Comment by forivall** [comment]
The latest `@types/workerpool` package released is 6.1.0, 5 months ago. If there are new features missing, please submit an issue to add those or feel free to submit a PR.
